src/main/java/io/confluent/idesidecar/restapi/application/ProxyProcessorBeanProducers.java

```diff
@@ -101,13 +102,28 @@ public Processor<ProxyContext, Future<ProxyContext>> RbacProxyProcessor(
 
   @Produces
   @Singleton
-  @Named("CCloudProxyProcessor")
-  public Processor<ProxyContext, Future<ProxyContext>> ccloudProxyProcessor(
-      ControlPlaneProxyProcessor genericProxyProcessor
+  @Named("controlPlaneProxyProcessor")
+  public Processor<ProxyContext, Future<ProxyContext>> ccloudProxyControlPlaneProcessor(
+      ControlPlaneProxyProcessor controlPlaneProxyProcessor
   ) {
     return Processor.chain(
         new ConnectionProcessor<>(connectionStateManager),
-        genericProxyProcessor,
+        controlPlaneProxyProcessor,
+        cCloudApiAuthProcessor,
+        new ProxyRequestProcessor(webClientFactory, vertx),
+        emptyProcessorProxyContext
+    );
+  }
+
+  @Produces
+  @Singleton
+  @Named("dataPlaneProxyProcessor")
+  public Processor<ProxyContext, Future<ProxyContext>> ccloudDataPlaneProxyProcessor(
```

Comment:
This proxy is specific to Flink and does not support proxying, for instance, Kafka data plane routes. We should reflect this in namings.